servers/cu/src/config.js

```diff
@@ -144,7 +145,8 @@ const CONFIG_ENVS = {
     PROCESS_CHECKPOINT_FILE_DIRECTORY: process.env.PROCESS_CHECKPOINT_FILE_DIRECTORY || tmpdir(),
     PROCESS_MEMORY_CACHE_MAX_SIZE: process.env.PROCESS_MEMORY_CACHE_MAX_SIZE || 500_000_000, // 500MB
     PROCESS_MEMORY_CACHE_TTL: process.env.PROCESS_MEMORY_CACHE_TTL || ms('24h'),
-    BUSY_THRESHOLD: process.env.BUSY_THRESHOLD || 0 // disabled
+    BUSY_THRESHOLD: process.env.BUSY_THRESHOLD || 0, // disabled
+    RESTRICT_TO_PROCESSES: process.env.RESTRICT_TO_PROCESSES ? process.env.RESTRICT_TO_PROCESSES.split(',') : []
```

Comment:
By using `commaDelimitedArraySchema` in the `domainConfigSchema`, this guard and parsing can be pushed down into the domain.

Then this line can just be:
```
RESTRICT_TO_PROCESSES: process.env.RESTRICT_TO_PROCESSES || []
```